### PR TITLE
CIF-1114 - Resizing Venia image does not work

### PR DESCRIPTION
**Pull request by herzog31**
## Description

* All WCM core component clientlibs were missing, since they are not automatically added by the components.
* This implementation is similar to the AEM project archetype:
    * https://github.com/adobe/aem-project-archetype/tree/master/src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/clientlib-base
   * https://github.com/adobe/aem-project-archetype/tree/master/src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/components/structure/page

## How Has This Been Tested?

* Manually tested the WCM components in the Sites editor.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [X] All unit tests pass on CircleCi.
- [X] I ran all tests locally and they pass.

### CODE REVIEW

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/clientlib-base/css.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}
```
Comment (mhaack) [inline src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/clientlib-base/css.txt]: I hash sign a problem with mvn archetype so we need to encode it?

**Review by mhaack — APPROVED** [review]

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/clientlib-base/css.txt (hunk 1 above)
Comment (herzog31) [reply src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/clientlib-base/css.txt]: The archetype uses hashes to indicate variables, so yes.

**Review by dplaton — APPROVED** [review]

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/clientlib-base/css.txt (hunk 1 above)
Comment (cjelger) [reply src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/clientlib-base/css.txt]: If the file is anyway empty, let's remove that horrible header ... or even remove the file? (not sure if it's ok to not have that file in a clientlib)

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/clientlib-base/js.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}
+${hash} Copyright 2019 Adobe Systems Incorporated
```
Comment (cjelger) [inline src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/clientlib-base/js.txt]: If the file is anyway empty, let's remove that horrible header ... or even remove the file? (not sure if it's ok to not have that file in a clientlib)

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/clientlib-base/css.txt (hunk 1 above)
Comment (herzog31) [reply src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/clientlib-base/css.txt]: I double checked and the files can be empty and the clientlib is still properly generated. So I removed the ugly header.

Comment (mhaack) [reply src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/clientlib-base/css.txt]: 👍

**Review by cjelger — APPROVED** [review]

**Pull request merged**